src/Data/ArrayBuffer/Typed.js

```diff
@@ -1,75 +1,185 @@
 "use strict";
 
-// module Data.ArrayBuffer.Typed
 
-exports.asInt8Array = function(v) {
-  return new Int8Array(v.buffer, v.byteOffset, v.byteLength);
-}
+exports.polyFill = function polyFill () {
```

Comment:
Why do you export this? Would it be problematic to just ejecute the polyfill directly?

Reply:
I don't think it would, but I was just trying to be on the safe side - it technically causes an effect, and although I know ffi files only get executed once, I just felt like someone might want finer granularity of control. But honestly I think it would be fine to just execute - I pulled it [from MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#Methods_Polyfill).